### ISSUE TITLE
targets issue

### ISSUE THREAD

**Issue by Jaekyumkim** [yhenon/pytorch-retinanet#19]
hi i trained your retinanet code.

So it works well. but i have one question. 

You divide the regression targets by [0.1, 0.1, 0.2, 0.2] in loss function.

But i can not understand why you did like this.

Can you answer me that question?

**Comment by yhenon** [comment]
I see you asked a similar question here: https://github.com/fizyr/keras-retinanet/issues/659

I think @hgaiser 's answer is quite good, so I'll go ahead and close this.
